tests/integration/mocks/contacts.js

```diff
@@ -3,16 +3,32 @@ var mock = {}
 mock = {
     contact_id: "place",
     contact : {
-        name: "dave",
-        vat_number: "456ghtd",
-        city: "london",
-        telephone: "0203467893"
+        mainObject: {
```

Comment:
because I have re-used the smae methods for splitting objects so I was just thinking of property names that could work for both orders/units as well as contacts/reminders.